### ISSUE TITLE
DGP 3: Proposal for using coinvoting for all decisions apart from proposals selection

### ISSUE THREAD

**Issue by parag-arora** [district0x/governance#3]
Had posted this in proposals repository but was suggested this is a better place to vote. I propose to use [coinvoting.github.io](https://coinvoting.github.io) which I have developed to create any proposal coin voting backed by DNT tokens (or any ERC20 tokens once districts opt to have their own tokens). Coinvoting support HTML as well for description and options. 

Examples of such proposals can be intermittent decisions regarding other districts or in governence. I will be maintaining coinvoting to make sure the support is always provided.

**Comment by lkngtn** [comment]
I like the thought but I do not think this makes sense to set the expectation that this would be used for decision making. 

If the intention is for it to simply provide feedback to the core team then I think it's useful and sufficient for that purpose. However, if the intention is to truly decentralize the governance of the project we need to also consider how to ensure that enough of the population of DNT holders are represented in every decision that affects them and that decisions have significant visibility and are actively discussed prior to an official decision. 

I think there is a lot of utility in allowing DNT users to post polls on any topic, however, unless we have a mechanism to ensure adequate participation and representation it would be unwise to put too much stake in the outcome of these polls.

**Comment by parag-arora** [comment]
I agree. Engagement is a problem and we have seen in most developed nations of the world. People want to vote once the decision is taken. 
I will think on this problem. Thanks @lkngtn . Will have a great solution soon.
